<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagDataAnalyzer</class>
 <widget class="QDockWidget" name="DiagDataAnalyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>1277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Analyzer</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Classification</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="buttonLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QToolButton" name="triangleButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add Triangle</string>
           </property>
           <property name="icon">
            <iconset resource="xvrresource.qrc">
             <normaloff>:/primitive/resources/primitives/triangle.png</normaloff>:/primitive/resources/primitives/triangle.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="quadButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add Quad</string>
           </property>
           <property name="icon">
            <iconset resource="xvrresource.qrc">
             <normaloff>:/primitive/resources/primitives/quad.png</normaloff>:/primitive/resources/primitives/quad.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="bananaButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add Banana</string>
           </property>
           <property name="icon">
            <iconset resource="xvrresource.qrc">
             <normaloff>:/primitive/resources/primitives/banana.png</normaloff>:/primitive/resources/primitives/banana.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonInputFileName">
        <property name="text">
         <string>Input File Names...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditInputFileName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonRead">
        <property name="text">
         <string>Read</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxCurrentTime"/>
      </item>
      <item>
       <widget class="QLabel" name="labelTotalTimeStep">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>of 0000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonBackToFirst">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/VCR/resources/vcr/first.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPrev">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/VCR/resources/vcr/prev.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPlay">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="xvrresource.qrc">
          <normaloff>:/VCR/resources/vcr/play.png</normaloff>
          <normalon>:/VCR/resources/vcr/pause.png</normalon>
          <selectedoff>:/VCR/resources/vcr/pause.png</selectedoff>
          <selectedon>:/VCR/resources/vcr/play.png</selectedon>:/VCR/resources/vcr/play.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonNext">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/VCR/resources/vcr/next.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonGoToLast">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/VCR/resources/vcr/last.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonLoop">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/VCR/resources/vcr/loop.png</normalon>
          <selectedoff>:/VCR/resources/vcr/loop.png</selectedoff>
          <selectedon>:/VCR/resources/vcr/loop.png</selectedon>
         </iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>450</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Data Info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelExtentInfo">
         <property name="text">
          <string>Extent:  </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableWidgetVarsInfo">
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Selected</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Data Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Data Range</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="xvrresource.qrc"/>
 </resources>
 <connections/>
</ui>
